site_scons/site_tools/go_builder.py

```diff
@@ -8,7 +8,6 @@
 from SCons.Script import Configure, Exit, File, GetOption, Glob, Scanner
 
 GO_COMPILER = 'go'
-MIN_GO_VERSION = '1.18.0'
```

Comment:
May be overengineering, but in the interest of minimizing the number of places that need to be updated, maybe this could be based on the version in `go.mod`?

Incidentally, for the record, I do agree that we should keep this scons check for the clear failure when building from source with an outdated Go toolchain.

Reply:
I did this and it seems to work. Thanks, this should make things a little easier to maintain in the future.